core/cloudflow-operator/src/main/scala/cloudflow/operator/CloudflowOwnerReferences.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cloudflow.operator
+
+import skuber.OwnerReference
+
+case class CloudflowOwnerReferences(list: List[OwnerReference])
```

Comment:
There is only one owner right (the CR)? what would be wrong with using something like this instead?
```
case class CloudflowOwnerReference(owner: OwnerReference)
```

Reply:
The k8s concept is a list, so, in the end, you add a list to the K8s object, you could argue that the conversion could happen then instead of here.